airflow/macros/__init__.py

```diff
@@ -66,7 +67,7 @@ def ds_format(ds: str, input_format: str, output_format: str) -> str:
     return datetime.strptime(str(ds), input_format).strftime(output_format)
 
 
-def datetime_diff_for_humans(dt: Any, since: Optional[datetime] = None) -> str:
+def datetime_diff_for_humans(dt: Any, since: Optional[Union[datetime, DateTime]] = None) -> str:
```

Comment:
This should not be needed because `Union[datetime, DateTime]` is just `datetime`. This entire module should only need one change; see #20132.

Reply:
Ah yeah. I will remove macros changes then